Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -627,61 +633,56 @@ ExplicitBitVect *wrapRDKFingerprintMol(
   return res;
 }
 
+SparseIntVect<boost::uint64_t> *wrapUnfoldedRDKFingerprintMol(
```

Comment:
Yeah, there is unfortunately rather a lot of that here.

The whole code base is re-formatted as part of the clang-tidy branch, but that won't help master for a while.

I should probably do another bulk pass through master and re-run all the .h and .cpp files again